File: sub-model/src/main/java/org/github/base/mapper/ISysDeptMapper.java
package org.github.base.mapper;

import org.github.base.entity.SysDeptEntity;
import org.github.base.IMapper;
import org.github.mybatis.MyBatisMapper;

/**
 * <p>
 * 部门表 Mapper 接口
 * </p>
 *
 * @author JYD_XL
 * @since 2019-12-12
 */
@MyBatisMapper
public interface ISysDeptMapper extends IMapper<SysDeptEntity> {}

File: sub-model/src/main/java/org/github/base/mapper/ISysDeptMapper.kt
package org.github.base.mapper

import org.github.base.IMapper
import org.github.base.entity.SysDeptEntity
import org.github.mybatis.MyBatisMapper

/**
 * <p>
 * 部门表 Mapper 接口
 * </p>
 *
 * @author JYD_XL
 * @since 2019-11-19
 */
@MyBatisMapper
interface ISysDeptMapper: IMapper<SysDeptEntity>
